doc/BUILD.md

```diff
@@ -121,19 +121,59 @@ This file is meant to be renamed to __autoconf.ac__, possibly extended with
 extra commands.  In Metamath commands for strengthening compiler flags
 are added.
 
-### autoconf.ac
+### configure.ac
 
-This file encodes the features the OS needs to be tested for.  When processed by
-_autoconf_ a _configure_ script along with its input file _config.h.in_ is created.  
-Some instructions aim at replacing system dependent variables in _Makefile.am_,
-later used to support _make_.  The language used for encoding this is __M4__ 
-along with a couple of built-in commands of __autoconf__.  This language is 
-designed to provide cross-platform descriptions of features of the OS.
+This script file encodes the features the OS needs to be tested for in a
+portable way.  When processed by _autoconf_ a _configure_ script along with its
+input file _config.h.in_ is created.  A few instructions set project data like
+name or version number.  A couple of other instructions aim at patching system
+dependent variables in _Makefile.am_, later used to support _make_.
+
+The script language used for encoding this is __M4__ using a couple of built-in
+commands of __autoconf__.  This language is designed to provide cross-platform
+descriptions of features of the OS.
 
 ### autoconf
 
 This Unix program called __autoconf__, or its sibling __autoreconf__, is
-capable of generating a _configure_ shell script from the input _autoconf.ac_.
+capable of generating a _configure_ shell script from the input _configure.ac_.
+
+If you use _autoreconf_ this need not be called.
+
+### autoheader
+
+This program from _Autotools_ creates a _config.h.in_ from a _configure.ac_
+input file.
+
+If you use _autoreconf_ this is called automatically.
+
+### config.h.in
+
+This file is created by _autoheader_ and mostly contains a list of features the
+_configure_ tests the OS for.  The encoding is macro instructions in
+[C](https://en.wikipedia.org/wiki/C_(programming_language).  Its contents looks
```

Comment:
Unbalanced parens? Do the parens in the URL have to be percent-encoded?

Reply:
I will close this PR and open another one, fixing the unbalanced parens.  This became necessary because my repository is out of sync with Github after network problems.  This is for me the easieast way to fix that.